packages/ui/src/components/va-carousel/VaCarousel.stories.ts

```diff
@@ -1,12 +1,224 @@
-import { defineComponent } from 'vue'
```

Comment:
Please add a story with infinite for 0, 1, 2 slides. We can catch some edge cases there.